forch/forchestrator.py

```diff
@@ -230,10 +235,21 @@ def _process_static_device_behavior(self):
     def _reload_static_device_behavior(self, file_path):
         self._port_state_manager.clear_static_device_behaviors()
 
-        devices_state = yaml_proto(file_path, DevicesState)
+        try:
+            devices_state = yaml_proto(file_path, DevicesState)
+        except Exception as error:
+            msg = f'DVA disabled: could not load static behavior file {file_path}'
```

Comment:
Make sure there is a log message somewhere that indicates when this condition is cleared (the file appears).

Reply:
Done. Added logging below